core/src/components/loading/test/loading-id.spec.ts

```diff
@@ -1,10 +1,11 @@
 import { newSpecPage } from '@stencil/core/testing';
 
+import { Backdrop } from '../../backdrop/backdrop';
 import { Loading } from '../loading';
 
 it('loading should be assigned an incrementing id', async () => {
   const page = await newSpecPage({
-    components: [Loading],
+    components: [Loading, Backdrop],
```

Comment:
Backdrop was added to all affected unit tests (when the overlay is presented/dismissed), as the API for `.block()` and `.unblock()` causes an exception and a test failure if the component is not imported/defined.